### DISCUSSION TITLE
Theme options

### DISCUSSION THREAD

**Discussion by Nis3r** [mdn/mdn-community#561] [Content]
I hope this is the right place to post this if not please direct me to the right spot.
I've always had trouble reading black text on white backgrounds (or white text on black backgrounds) 
I'd love to see some more theme options or one where you can choose your own colours.
I'm trying to read through the accessibility guides and I'm finding them very hard to get through if I don't copy everything to a document with a background other than white. That solution usually messes up the formatting and caused other issues.

**Comment by MrBrain295** [comment]
Discussions have moved to [here](https://github.com/mdn/mdn-community/discussions).
